tests/utils/queryApiGateway.test.tsx

```diff
@@ -0,0 +1,150 @@
+import queryApiGateway, { buildApiUrl, extractJSON, Claim, QueryParams } from '../../utils/queryApiGateway'
+import mockEnv from 'mocked-env'
+import fs from 'fs'
+import fetch from 'node-fetch'
+
+// Mock some modules
+jest.mock('fs')
+jest.mock('node-fetch')
+// const { Response } = jest.requireActual('node-fetch')
+
+// Shared test constants
+const goodUrl = 'http://nowhere.com'
+
+/**
+ * Begin tests
+ */
+
+// Test queryApiGateway
+describe('Querying the API Gateway', () => {
+  const goodResponse = { ClaimType: 'PUA' }
+  const goodRequest = {
+    headers: {
+      id: '12345',
+    },
+  }
+
+  beforeEach(() => {
+    // Mock the fetch response
+    // fetch.mockReturnValue(Promise.resolve(new Response(JSON.stringify(goodResponse))))
```

Comment:
This line should be uncommented, but needs to pass linting before it can be commited